plugins/boltz/main.go

```diff
@@ -109,17 +123,24 @@ func NewPlugin(lnAPI agent_entities.NewAPICall, filter filter.FilteringInterface
 		if err != nil {
 			return nil, err
 		}
+		err = db.Insert(SecretDbKey, &Entropy{Data: entropy})
```

Comment:
looks like mnemonic setup could be extracted to new function. 
I would love to have NewPlugin() as clear as possible